src/App.tsx

```diff
@@ -1,34 +1,53 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useCallback, useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { useAppDispatch, useAppSelector } from './app/hooks';
+import { getTodos } from './api';
+import { actions as todoAction } from './features/todos';
 
 export const App: React.FC = () => {
+  const [loader, setLoader] = useState(true);
```

Comment:
```suggestion
  const [isLoading, setIsLoading] = useState(false);
```
This name is better for boolean state. also default value is false